marquez_client/version.py

```diff
@@ -0,0 +1,13 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+VERSION = '0.4.0' 
```

Comment:
Keeping two versions of `VERSION` can be problematic. One solution is to update the release script to also target this file. Then it will make the same update here as it does in setup.py. The other solution (which I think we discussed in our meeting) would be to import `VERSION` directly from `setup.py`.

Reply:
Agreed. Python does provide a guide on how to approach versioning using `version.py`, see https://packaging.python.org/guides/single-sourcing-package-version. I've opened an issue to address this #57